src/main/java/lotto/LottoRank.java

```diff
@@ -0,0 +1,44 @@
+package lotto;
+
+import java.util.Arrays;
+
+public enum LottoRank {
+    FIRST(6, 2_000_000_000),
+    SECOND(5, 30_000_000),
+    THIRD(5, 1_500_000),
+    FOURTH(4, 50_000),
+    FIFTH(3, 5_000),
+    OUT_OF_RANK(0, 0);
+
+    private static final int FIVE = 5;
+
+    private int countOfMatch;
+    private int price;
+
+    LottoRank(int countOfMatch, int price) {
+        this.countOfMatch = countOfMatch;
+        this.price = price;
+    }
+
+    public int getCountOfMatch() {
+        return countOfMatch;
+    }
+
+    public int getPrice() {
+        return price;
+    }
+
+    public static LottoRank valueOf(int countOfMatch, boolean bonusMatch) {
+        if (bonusMatch && countOfMatch == FIVE) {
+            return SECOND;
+        }
+        if (!bonusMatch && countOfMatch == FIVE) {
+            return THIRD;
+        }
```

Comment:
```suggestion
        if (bonusMatch && countOfMatch == SECOND.countOfMatch) {
            return SECOND;
        }
        if (!bonusMatch && countOfMatch == THIRD.countOfMatch) {
            return THIRD;
        }
```
로 작성하면 FIVE라는 상수는 제거 가능하겠네요. 
그리고 개인적으로 5 나 FIVE 의 차이는 없다고 생각합니다. FIVE를 봐도 5가 떠오르지 않나요?

Reply:
아 .. 이런방법이 있네요! 수정 해보았습니다 감사합니다 ㅎㅎ